src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -16,7 +16,8 @@ public HelpCommand() {}
     @Override
     public CommandResult execute() {
         return new CommandResult(
-                AddCommand.MESSAGE_USAGE
+                SortCommand.MESSAGE_USAGE
```

Comment:
This is good, make sure to update your `help` command as well, so that internally it also reflects the new commands to the user. However, it seems that you've removed `add` command from `help`, does that mean `add` functionality is removed from AB1?